masonite/providers/AppProvider.py

```diff
@@ -73,9 +73,9 @@ def boot(self, request: Request, route: Route):
 
     def _autoload(self, directories):
         Autoload(self.app).load(directories)
-    
+
     def _set_application_debug_level(self):
-        if self.app.make('Application').DEBUG == 'True':
-            self.app.make('Application').DEBUG == True
-        elif self.app.make('Application').DEBUG == 'False':
-            self.app.make('Application').DEBUG == False
+        if self.app.make('Application').DEBUG is 'True':
+            self.app.make('Application').DEBUG is True
```

Comment:
@josephmancuso Your opinion?

Reply:
or is that not true?

Reply:
nvm. That's not true. yeah i'm a fan of doing it like that

Reply:
This is what best explanation I can find. 
`is` checks that 2 arguments refer to the same object, `==` checks that 2 arguments have the same value.

So, I need to change that back to `==` for this line `if self.app.make('Application').DEBUG is 'True':`

Reply:
ok that's what I thought lol